src/platform/plugins/shared/console/test/scout/api/tests/spec_definitions.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import type { RoleApiCredentials } from '@kbn/scout';
```

Comment:
We have an automatic test discovery in place, that helps us to better balance CI runtime.
It requires test files name to match pattern `<your_test_case>.spec.ts`, so for the last CI run your tests wasn't discovered:

<img width="1565" height="590" alt="image" src="https://github.com/user-attachments/assets/3dcf3279-fcae-47e8-a8a9-43a2ae8a9c22" />